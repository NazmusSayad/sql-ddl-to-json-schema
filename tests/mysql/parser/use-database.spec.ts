import { getParsedFormat } from '../parse-handler';
import { run } from '../runner';

run(getParsedFormat, {
  'Parser: Should use test database': {
    queries: ['USE test;\nUSE `another_test`;'],
  },
});
