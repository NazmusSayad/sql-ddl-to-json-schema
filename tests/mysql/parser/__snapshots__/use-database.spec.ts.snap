// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parser: Should use test database 1`] = `
{
  "id": "MAIN",
  "def": [
    {
      "id": "P_DDS",
      "def": {
        "id": "P_USE_DB",
        "def": {
          "database": "test"
        }
      }
    },
    {
      "id": "P_DDS",
      "def": {
        "id": "P_USE_DB",
        "def": {
          "database": "another_test"
        }
      }
    }
  ]
}
`;
